fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/ContentObject.java

```diff
@@ -94,6 +94,11 @@ public FileObject getDescription() {
 		}
 	}
 	
+	@Override
+	public RepositoryObject getParent() {
+		return null;
```

Comment:
Might as well implement this, since it'll be the exact same as for a binary `return dataLoader.getParentObject(this);`